lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -49,12 +50,17 @@ export class PublicClientApplication implements IPublicClientApplication {
      * @param configuration {Configuration}
      */
     public static async createPublicClientApplication(
-        configuration: Configuration
+        configuration: Configuration,
+        featureSupportConfiguration?: FeatureSupportConfiguration
```

Comment:
Does this really need to be a separate object or can it be folded into the existing Configuration?

Reply:
I think long term vision is to have a feature flag config to be dynamically loaded to MSAL JS if possible. We can discuss, but separation helps in that context IMHO.

Reply:
Also rename to `featureConfig`

Reply:
Dynamic config is a bit orthogonal and I don't see how creating a separate object gets us closer to that goal. I'd like to see a design and get team agreement before we make public surface changes like this.

Reply:
We can merge with Config for now (I have some doubts here, expanded below). The objective is to feature gate this feature. One way of getting that in is to have feature control separated from Config, at least from user POV (which can be short-term controlled by MSAL JS before introducing it to public API). If we are adding it in config, removing it will become a breaking change. 

Of course we need a design and agreement before adding it at all places and making it a public API.

**TLDR:** Short term I am looking for controlling this feature with a flag. Let us sync offline to arrive at a common solution here.